src/examples/container/ContainerExamples.tsx

```diff
@@ -481,15 +481,15 @@ export default function ContainerExamples() {
             </GoAFormItem>
 
             <GoAFormItem label="Reason to deny">
-              <GoADropdown name="reason" onChange={() => {}}>
+              <GoADropdown name="reason" onChange={() => {}} width="35ch">
```

Comment:
You don't want to hardcode this value as it will cause issues on smaller screens.
![image](https://github.com/GovAlta/ui-components-docs/assets/41582/5f048194-fa37-4d76-8004-314d8cda4f0b)